src/ICeWall/doc/load_library.md

```diff
@@ -0,0 +1,12 @@
+## ICeWall load_library
+### Synopsis
+```
+  % ICeWall load_library <library_file_name>
+```
+### Description
+Additional information about the padcells in the library, over and above what is available in standard EDA views is specified in a library definitions file. This command loads this library data so that it can be used to build a padring.
+### Example
```

Comment:
Maybe reference back to the main README.md where you discuss how this file should be structured.